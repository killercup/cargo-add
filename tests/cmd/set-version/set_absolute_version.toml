bin.name = "cargo-set-version"
args = ["set-version", "2.0.0"]
status = "success"
stdout = ""
stderr = """
    Upgraded sample from 0.1.0 to 2.0.0
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
