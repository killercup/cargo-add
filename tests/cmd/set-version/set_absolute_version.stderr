    Upgraded sample from 0.1.0 to 2.0.0
